packages/playwright-core/src/server/page.ts

```diff
@@ -718,12 +726,12 @@ export class PageBinding {
   constructor(name: string, playwrightFunction: frames.FunctionWithSource, needsHandle: boolean) {
     this.name = name;
     this.playwrightFunction = playwrightFunction;
-    this.source = `(${addPageBinding.toString()})(${JSON.stringify(name)}, ${needsHandle})`;
+    this.source = `(${addPageBinding.toString()})(${JSON.stringify(name)}, ${needsHandle}, (${source})(true))`;
```

Comment:
Why can't we send `(${source})(true)` once into the page?

Reply:
Same applies to addPageBinding.